Sources/GRPC/AsyncAwaitSupport/GRPCAsyncBidirectionalStreamingCall.swift

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2021, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import _NIOConcurrency
+import Logging
+import NIOCore
+import NIOHPACK
+import NIOHTTP2
+
+#if compiler(>=5.5)
```

Comment:
Roger that. I followed what I saw in NIO where I assumed it was after the imports for a good reason 😅 

https://github.com/apple/swift-nio/blob/main/Sources/_NIOConcurrency/AsyncAwaitSupport.swift#L17

Happy to move all the guards so that the file is completely empty if they are not satisfied.

Reply:
Addressed in 8babddce.